### DISCUSSION TITLE
posts/autonomous-3/

### DISCUSSION THREAD

**Discussion by giscus** [seyeon040768/seyeon040768.github.io#10] [Comments]
# posts/autonomous-3/

참고사항

https://blog.seyeon-lee.com/posts/autonomous-3/

**Comment by minmaxHong** [comment]
초반에 글 적어주신, 변환행렬에서 T 와 R은 스칼라인가요? 아님 벡터인가요?

**Reply by seyeon040768** [reply]
3차원 기준으로 $\mathbf{t}$는 크기가 3인 벡터이고 $\mathbf{R}$은 3*3 행렬입니다!

**Comment by KIM-WAN-TAE** [comment]
Rodrigues’ rotation formula(로드리게스 회전 공식)이 공식에서 사용자 지정 변경 값과 이미 정해져있는 값이 각각 무엇인지 궁금합니다.

**Reply by seyeon040768** [reply]
회전축 $\mathbf{n}$과 회전시킬 벡터 $\mathbf{v}$가 사용자가 지정할 수 있는 값입니다.
하지만 대부분 $\mathbf{n}$은 기저 변환을 위해 $x$, $y$, $z$축을 사용합니다.

**Comment by jongmyeongpark** [comment]
이차원 에서 점의 이동을 행렬로 표현할 때 x',y',1 로 표현이 되는데 이때 1이 의미하는 바가 있나요? 
좋은 글 잘 읽었습니다.

**Reply by seyeon040768** [reply]
1은 동차좌표계로 이동을 할 때 더할 상수항을 위해 사용합니다.
